.gitignore

```diff
@@ -8,6 +8,7 @@ obj/
 build/
```

Comment:
You can remove most of the above and only keep bin, debug and build.

Reply:
Done